docs/sources/writing-guide/front-matter/index.md

```diff
@@ -101,15 +112,50 @@ labels:
     - enterprise
 ```
 
-For a **directory of pages** that describe a feature only available in Grafana Cloud, the branch bundle `_index.md` file front matter should include the following:
+#### Set cascading front matter
+
+When a directory of pages share the same front matter, it is convenient to set them all at once with cascading front matter.
+
+There are two forms of cascading front matter.
+The first is a `cascade` map of keys to values.
+The second is a sequence of mappings with the `_target` keyword.
+You cannot override the latter with the former.
+For more information about cascading front matter, refer to [Front Matter Cascade](https://gohugo.io/content-management/front-matter/#front-matter-cascade).
+
+{{% admonition type="note" %}}
+To avoid confusing behavior, set all cascading front matter with the sequence of mappings form.
+{{% /admonition %}}
+
+For a **directory of pages** under `/docs/grafana/latest/troubleshooting/` that describe a feature only available in Grafana Cloud, the branch bundle `_index.md` file front matter should include the following:
 
 ```yaml
 cascade:
-  labels:
-    products:
-      - cloud
+  - _target:
+      path: "/docs/grafana/*/troubleshooting/**"
+    labels:
+      products:
+        - cloud
 ```
 
+In the preceding example, the `cascade` value is a YAML block sequence of length one, as denoted by the single dash and space (`- `).
+
+The value of the only element in `cascade` sequence is a mapping with two keys: the Hugo reserved `_target`, and the Grafana specific `labels`.
+
+The `_target` key is a map of the key `path` to the value `"/docs/grafana/*/troubleshooting/**"`.
+For more information about the the `path` value, refer to [Specify a path](#specify-a-path).
+
+The `labels` key is the same mapping that is used for individual pages.
+
+##### Specify a path
+
+The value of the `_target.path` mapping is a string matching paths from the Hugo content root.
+Glob patterns specified with `*` and `**` match multiple paths.
+
+The `*` pattern matches any file or directory.
+It is useful for matching any version directory for versioned documentation.
+
+The `**` pattern matches all directories and files recursively.
+
```

Comment:
So I vaguely understand this, but this is much more conceptual than instructional and I think it needs to be a bit more instructional. The concepts are probably important, but ultimately someone comes to this to get a job done and maybe it's best to just tell someone "how" rather than "what." Let's sync up on this.

Reply:
I agree with @imatwawana 's comment. I'll have to study this doc for a bit when needed. Would an instructional video be totally nuts here? Doesn't have to be super polished.